wsrep_loader.c

```diff
@@ -225,8 +225,10 @@ void wsrep_unload(wsrep_t *hptr)
     } else {
         if (hptr->free)
             hptr->free(hptr);
+#ifndef USE_VALGRIND
```

Comment:
Maybe it would be good to add a comment why we are not unloading the so?

Reply:
The other similar ifndefs have no comments, and I explained it in the commit message - so git blame shows the reasoning. It's also a common pattern for valgrind, nothing special.